### DISCUSSION TITLE
Crimson exploit?

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1243] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I saw this exploit called Crimson here (https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/Crimson%20(Corrupt%20Extensions), it seems to corrupt extensions, so does it work on v129?

**Comment by Sam-hacks1** [comment]
I'ma try it on v130

**Reply by Sam-hacks1** [reply, marked as answer]
doesn't work
